projects/packages/tracking/src/class-tracking.php

```diff
@@ -47,6 +47,12 @@ public function __construct( $product_name = 'jetpack', $connection = null ) {
 			// TODO We should always pass a Connection.
 			$this->connection = new Connection\Manager();
 		}
+
+		if ( ! did_action( 'jetpack_set_tracks_ajax_hook' ) ) {
+			add_action( 'wp_ajax_jetpack_tracks', array( $this, 'ajax_tracks' ) );
+
+			do_action( 'jetpack_set_tracks_ajax_hook' );
```

Comment:
This new action needs a [docblock](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters). I can add that if you'd like.

I don't think I should provide the approval for this PR, so we'll need to get someone else from Crew to approve it.

Reply:
I had to update a `composer.lock` file for this PR, so I took the opportunity to add the docblock in commit 2aeeb922